### ISSUE TITLE
unclear doc entry for `mode` in `Pkg.status()`

### ISSUE THREAD

**Issue by lmiq** [JuliaLang/Pkg.jl#3241]
The help entry for `Pkg.status` prints:

```julia
help?> Pkg.status
  Pkg.status([pkgs...]; mode::PackageMode=PKGMODE_PROJECT, diff::Bool=false, compat::Bool=false, io::IO=stdout)

  Print out the status of the project/manifest.
...
  If mode is PKGMODE_PROJECT, print out status only about the packages that
  are in the project (explicitly added). If mode is PKGMODE_MANIFEST, print
  status also about those in the manifest (recursive dependencies). If there
  are any packages listed as arguments, the output will be limited to those
  packages.

```

It is not clear what the user must do to set `mode`. For instance, none of these work:

```julia
julia> a = Pkg.status(mode=Pkg.PackageMode(PKGMODE_PROJECT()))
ERROR: UndefVarError: PKGMODE_PROJECT not defined
Stacktrace:
 [1] top-level scope
   @ REPL[16]:1

julia> a = Pkg.status(mode=Pkg.PackageMode(PKGMODE_PROJECT))
ERROR: UndefVarError: PKGMODE_PROJECT not defined
Stacktrace:
 [1] top-level scope
   @ REPL[13]:1

julia> a = Pkg.status(mode=Pkg.PackageMode("PKGMODE_PROJECT"))
ERROR: MethodError: no method matching Pkg.Types.PackageMode(::String)
Closest candidates are:
  Pkg.Types.PackageMode(::Integer) at Enums.jl:197
Stacktrace:
 [1] top-level scope
   @ REPL[14]:1
```

In the last one one gets the hint that `PackageMode` requires an integer as input, but that does not correspond to anything in the help entry.

**Comment by fredrikekre** [comment]
It is literally `PKGMODE_PROJECT`/`PKGMODE_MANIFEST`:
```
julia> Pkg.status(mode=PKGMODE_PROJECT)
Status `~/.julia/environments/v1/Project.toml`
[...]
```
Not sure why you where trying a bunch of other things than what the document says but if this is not clear it can of course be updated.

**Comment by lmiq** [comment]
Oh, I see. What happened is that I tried:

```
julia> Pkg.status(mode=PKGMODE_PROJECT)
ERROR: UndefVarError: PKGMODE_PROJECT not defined
Stacktrace:
 [1] top-level scope
   @ REPL[9]:1
```

and then started to try random things, but that is all because generally with `Pkg` we use `import Pkg`, and not `using`, and since the first try didn't work I started to sort of speculate if `PKGMODE_PROJECT` was a function, a constructor, etc.

Not sure if something can be improved, except perhaps put an example.
